packages/rs-drive/src/drive/batch/drive_op_batch.rs

```diff
@@ -522,6 +528,12 @@ impl DriveOperationConverter for DocumentOperationType<'_> {
                                 storage_flags,
                             } = update_operation;
 
+                            let document_cbor = document.to_cbor();
+
+                            // If serialized document not submitted updates are not performed
+                            // TODO: figure that out
+                            let serialized_document = serialized_document.or(Some(&document_cbor));
```

Comment:
@QuantumExplorer we probably need to fix it properly

Reply:
No this is just wrong

Reply:
@QuantumExplorer the thing is `packages/rs-drive/src/drive/document/update.rs` ignores `DocumentRefWithoutSerialization` variant and only uses `DocumentRefAndSerialization`.  But since we have this code `let document_info =
                                if let Some(serialized_document) = serialized_document {
                                    DocumentRefAndSerialization((
                                        document,
                                        serialized_document,
                                        storage_flags,
                                    ))
                                } else {
                                    DocumentRefWithoutSerialization((document, storage_flags))
                                };` and not passing serialized document we get only `DocumentRefWithoutSerialization`